### ISSUE TITLE
Fish has stopped suggesting autocompletions

### ISSUE THREAD

**Issue by matterhorn103** [fish-shell/fish-shell#10602]
fish version = 3.7.1
OS: Linux (6.9.7) x86_64
Distro: OpenSUSE Tumbleweed-Slowroll 20240605
DE: KDE Plasma 6.0.5
Terminal: Konsole 24.05.1

Fish has stopped suggesting me any completions. Nothing is shown and pressing the forward arrow key does nothing, even when I know I have recently performed a matching command in the same directory. Even immediately after running a command, that command doesn't get suggested.

History is still there, and I can use the partial history matching using the up and down arrow keys like normal.

`~/.local/share/fish/generated_completions/` has plenty in it. `fish_complete_path` has several sensible directories in it.

I really have no idea what could have changed to make fish break like this. Any ideas?

Slowroll and Tumbleweed update fairly frequently so I don't know if it has to do with that. I have another Tumbleweed install at home, I'll check later if the issue is the same there.

The only other thing that has recently changed is that I set `XDG_DATA_HOME` etc., which were before not defined, but I just set them to the recommended defaults which is where fish was putting its directories anyway.

**Comment by ridiculousfish** [comment]
Can you verify that the variable `fish_autosuggestion_enabled` is not set? Does this reproduce if you invoke fish like `HOME=(mktemp -d) fish`?

**Comment by matterhorn103** [comment]
I will have to wait till Monday to check on my work PC.

I can confirm though that the issue is exactly the same on my PC at home, which is running normal Tumbleweed (snapshot 20240704, so a month ahead with updates vs the work machine, on Plasma 6.1.2 but with the same version of Konsole).

On this home PC at least, `fish_autosuggestion_enabled` is not set. And the issue remains after trying your suggestion.

**Comment by matterhorn103** [comment]
Ah ok, I've found the problem. It's because I changed my terminal colour scheme (to [one half dark](https://github.com/sonph/onehalf)), and clearly in this scheme the same colour is used for the autosuggestions as the background, rendering the text invisible. And indeed what I said here:

> and pressing the forward arrow key does nothing, even when I know I have recently performed a matching command in the same directory.

seems not to be true after all.

Which of the ANSI colours is used for the autosuggestion colour? It seems to be whatever Konsole lists as the "Intense" version of "Colour 1", which is presumably ANSI "bright black" with the foreground code of `90`.

Are there any conventions around the use of the "bright" terminal colours? The One Half Dark scheme just uses the same colours for normal/bright/faint of each of the 10 colours. Perhaps I can suggest to the colour scheme's maintainers that they are made to vary.
